kpatch-build/kpatch-build

```diff
@@ -906,14 +921,9 @@ elif [[ -e "$KERNEL_SRCDIR"/.config ]] && [[ -e "$VERSIONFILE" ]] && [[ "$(cat "
 	echo "Using cache at $KERNEL_SRCDIR"
 
 else
-	if [[ "$DISTRO" = fedora ]] || [[ "$DISTRO" = rhel ]] || [[ "$DISTRO" = ol ]] || [[ "$DISTRO" = centos ]] || [[ "$DISTRO" = openEuler ]] || [[ "$DISTRO" = photon ]]; then
-
-		[[ "$DISTRO" = fedora ]] && echo "Fedora distribution detected"
-		[[ "$DISTRO" = rhel ]] && echo "RHEL distribution detected"
-		[[ "$DISTRO" = ol ]] && echo "Oracle Linux distribution detected"
-		[[ "$DISTRO" = centos ]] && echo "CentOS distribution detected"
-		[[ "$DISTRO" = openEuler ]] && echo "OpenEuler distribution detected"
-		[[ "$DISTRO" = photon ]] && echo "Photon OS distribution detected"
+	if is_supported_distro "$DISTRO"; then
```

Comment:
The following code only applies to RPM-based distros.  So this will break debian/ubuntu.

Reply:
oh~~I apologize for my carelessness! The previous code will break the judgment of debian/ubuntu.
Now I try an other way to fix this problem.
I declear a new list of SUPPORTED_RPM_DISTRO which exclude debian and ubunto while SUPPORTED_DISTRO contains all the supported distrobution.
Then, we just need to judge if this release is RPM-base or not.
For the print function, it can apply to all release, I think.

Please check my newer version of the commit. Thanks~~ :)